ansible/group_vars/all.yml

```diff
@@ -5,6 +5,8 @@
 rhevm_username: admin@internal
 rhevm_password: smartvm
 export_datastore: export
+rhevm_host: performance-testing-domain.com
```

Comment:
Lets move this above rhevm_username to match the style of the other hosts that are configured in this file.